### PR TITLE
[docs] Describe the debugging cli option

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs @helen-dikareva

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -299,6 +300,14 @@ testcafe chrome my-tests --app 'node server.js'
 
 Use the [--app-init-delay](#--app-init-delay-ms) option to specify the amount of time allowed for this command to initialize the tested application.
 
+### -d, --debug-mode
+
+Specify this option to run tests in the debugging mode. In this mode, test execution is paused before the first action or assertion allowing you to invoke the developer tools and debug.
+
+In the footer, a status bar is displayed in which you can resume test execution or step to the next action or assertion.
```
Comment (helen-dikareva) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: There is `The footer displays buttons that allow you to continue test execution or step to the next test action.` in our [`Debugging` section](https://devexpress.github.io/testcafe/documentation/test-api/debugging.html#client-side-debugging). No word about assertions. May be we need to add this?

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 2]
```diff
@@ -299,6 +300,14 @@ testcafe chrome my-tests --app 'node server.js'
 
 Use the [--app-init-delay](#--app-init-delay-ms) option to specify the amount of time allowed for this command to initialize the tested application.
 
+### -d, --debug-mode
+
+Specify this option to run tests in the debugging mode. In this mode, test execution is paused before the first action or assertion allowing you to invoke the developer tools and debug.
```
Comment (kirovboris) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: Will it work for `beforeEach` actions?

Comment (helen-dikareva) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: Yep, it works for `test.before`/ `test.after` and `fixture.beforeEach`/`fixture.afterEach`

Comment (kirovboris) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: In this case we should describe it in docs.

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
Ping @helen-dikareva

**Pull request merged**